wdio.conf.js

```diff
@@ -93,7 +93,11 @@ exports.config = {
               'dom.webnotifications.enabled': false,
               'media.getusermedia.screensharing.enabled': true,
               'media.navigator.permission.disabled': true,
-              'media.navigator.streams.fake': true
+              'media.navigator.streams.fake': true,
```

Comment:
These kind of changes should be in their own PR with a description of why they were made.